src/lib/firebase/controller.js

```diff
@@ -21,3 +27,15 @@ export const createUserDocument = async (userId, userData) => {
         // console.error("Error creating user document:", error);
     }
 };
+
+//function responsible for updating profile info
+export const updateUserDocument = async (userId, userData) => {
```

Comment:
does the function have to be asynchronously executed  ??

Reply:
not really, it only makes it executable at any time therefor i always go with asyc if i see a chance since asyc is recommanded for faster excustion when you have multiple function that are going to be working at the same time